libs/iblprefilter/src/IBLPrefilterContext.cpp

```diff
@@ -158,6 +160,11 @@ IBLPrefilterContext& IBLPrefilterContext::operator=(IBLPrefilterContext&& rhs) n
     return *this;
 }
 
+void IBLPrefilterContext::setCubemapFaceResolution(unsigned int res)
+{
```

Comment:
super nit (both formatting and param type):
```suggestion
void IBLPrefilterContext::setCubemapFaceResolution(uint32_t res) {
```

Reply:
Fixed in 02830cd